recommendation/tools.md

```diff
@@ -84,5 +84,10 @@ and the scope may identify a particular stream of development that that tool is
       <td>Jest</td>
       <td><a href="https://jestjs.io/" rel="nofollow">https://jestjs.io/</a></td>
     </tr>
+    <tr>
+      <td>E2E testing</td>
+      <td>Selenium</td>
```

Comment:
I'm not sold on Selenium as "the recommendation" just because it is bulky and costs a lot of resources to run. It might be best to run this recommendation by our AppDev-SA.

@Path125 @CalvinRodo

Reply:
Selenium is really the only multi-browser option that you have available it has some nice libraries for interacting with it in pretty much every language out there but it still sucks compared to say Cypress which is much nicer, I prefer a hybrid approach Cypress for e2e and selenium for basic smoke tests in each browser.